src/scripts/main.js

```diff
@@ -2,7 +2,25 @@
 
 const people = require('./lib/people');
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+const table = document.querySelector('tbody');
 
-// write your code here
+people.forEach(element => {
+  const {
+    sex,
+    born,
+    died,
+    age = died - born,
+    century = Math.ceil(died / 100),
+  } = element;
+
+  table.insertAdjacentHTML('beforeend', `
+    <tr>
+    // <td>${element.name}</td> 
+    <td>${sex === 'm' ? 'Male' : 'Female'}</td>
+    <td>${born}</td>
+    <td>${died}</td>
+    <td>${age}</td>
+    <td>${century}</td>
+  </tr>
```

Comment:
Also notice that you didn't make code formatting from this comment: https://github.com/mate-academy/js_task_generate_table_DOM/pull/680#discussion_r1108272517